### DISCUSSION TITLE
Can't access Windows Defender/Security as an Admin

### DISCUSSION THREAD

**Discussion by qf0** [HotCakeX/Harden-Windows-Security#276] [Q&A]
I can't view Windows Security or Defender after using this script. The hardening is also blocking PowerShell from installing OP's Cloudflare DNS over HTTPS etc. I am under an Admin account and tried under a usermode account too.

**Comment by HotCakeX** [comment]
There is nothing in the script that prevents any of them from happening. I need some screenshots to see what you mean.

**Reply by qf0** [reply]
A few restarts later and I'm able to access it again randomly. I'm unsure what caused it, but Windows Security window wouldn't even open or just be a black screen with Windows Security in the title bar.

**Reply by HotCakeX** [reply]
Hmm, never seen that behavior before, must've been caused most likely by something else, glad it's working now though
